docs/resources/vault_cluster.md

```diff
@@ -94,6 +94,16 @@ Optional:
 - `grafana_endpoint` (String) Grafana endpoint for streaming audit logs
 - `grafana_password` (String, Sensitive) Grafana password for streaming audit logs
 - `grafana_user` (String) Grafana user for streaming audit logs
+- `http_basic_password` (String, Sensitive) HTTP basic authentication password for streaming audit logs
+- `http_basic_user` (String) HTTP basic authentication username for streaming audit logs
+- `http_bearer_token` (String, Sensitive) HTTP bearer authentication token for streaming audit logs
```

Comment:
I think we should highlight in the description how these work, like if you have user you need password, and if you have either of them you can't have token